lib/jsonld_serializers.js

```diff
@@ -104,7 +104,7 @@ class JsonLdItemSerializer extends JsonLdSerializer {
       // If it's a numeric, force it to appear as a single val
       var singleValue = val && val.length === 1 && (typeof val[0]) === 'number'
       // Properties with '_' exist for specialized querying; don't save them to object:
-      if (indexedProperty.indexOf('_') < 0 && this.body[indexedProperty]) {
+      if (indexedProperty.indexOf('_') < 0 && this.body[indexedProperty] || typeof this.body[indexedProperty] === 'boolean') {
```

Comment:
Careful with operator precedence here. `||` has lower precedence to `&&`, so this actually evaluates to `(indexedProperty.indexOf('_') < 0 && this.body[indexedProperty]) || typeof this.body[indexedProperty] === 'boolean' (when we technically want the boolean type to be an exception to the the truthyness check)